src/test/java/bio/terra/service/snapshotbuilder/query/QueryTest.java

```diff
@@ -64,15 +64,16 @@ void renderSQLWithLimit(CloudPlatform platform) {
     String actual = createQueryWithLimit().renderSQL(CloudPlatformWrapper.of(platform));
     String expected = "SELECT t.* FROM table AS t";
     if (cloudPlatformWrapper.isAzure()) {
-      assertThat(actual, is("TOP 25 " + expected));
+      assertThat(actual, is("SELECT TOP 25 t.* FROM table AS t"));
```

Comment:
Should we use the `equalToCompressingWhiteSpace `? Since we're doing a strong compare? Also, do you think it would be worth adding the string value describing the assert to the start?